vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -2,7 +2,7 @@
 
 #nowarn "40"
 
-namespace Microsoft.VisualStudio.FSharp.ProjectSystem 
+namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem 
```

Comment:
I like how this is being dogfooded right in the same checkin, without the need to even have last known good carrying the feature :+1: